roles/extract_operator_bundle/tasks/main.yml

```diff
@@ -0,0 +1,42 @@
+- name: "Ensure that the operator bundle directory is empty"
+  file:
+    state: "absent"
+    path: "{{ operator_bundle_dir }}"
+  when: operator_bundle_dir is defined
+
+- name: "Ensure that the operator testing directory exists and is empty"
+  file:
+    state: "{{ item }}"
+    path: "{{ operator_work_dir }}"
+  with_items:
+    - absent
+    - directory
+  when: operator_work_dir is defined
+
+- name: "Copy the bundle image layers into a local directory using skopeo"
+  shell: "skopeo copy docker://{{ bundle_image }} dir:{{ operator_bundle_dir }} -f oci"
```

Comment:
We can directly copy image to `oci` format
skopeo copy docker://{{ bundle_image }} oci:{{ operator_bundle_dir }}

Reply:
Changed.